core/src/voice/sampler.rs

```diff
@@ -148,33 +148,43 @@ impl<S: Simd, Sampler: BufferSampler> SIMDSampleGrabber<S> for SIMDSampleGrabber
 
 // Sampler generator
 
-pub struct SIMDStereoVoiceSampler<S, Pitch, Grabber>
+pub struct SIMDStereoVoiceSampler<S, Pitch, Cutoff, Grabber>
 where
     S: Simd,
     Pitch: SIMDVoiceGenerator<S, SIMDSampleMono<S>>,
+    Cutoff: SIMDVoiceGenerator<S, SIMDSampleMono<S>>,
     Grabber: SIMDSampleGrabber<S>,
 {
     grabber_left: Grabber,
     grabber_right: Grabber,
 
     pitch_gen: Pitch,
 
+    cutoff: Cutoff,
```

Comment:
It's probably ok to remove this cutoff code, if making a cutoff filter this wouldn't exactly be the way to do it. I can show you how to ideally approach it later. I'm pretty sure this is unused anyway so can remove

Reply:
Yeah it is unused currently. I'll remove it.